@here/harp-mapview/lib/geometry/TileGeometryCreator.ts

```diff
@@ -1272,7 +1272,10 @@ export class TileGeometryCreator {
                         ? getPropertyValue(technique.opacity, Math.floor(zoomLevel))
                         : defaultRenderParams.opacity,
                 backgroundOpacity:
-                    technique.backgroundOpacity !== undefined
+                    technique.backgroundSize !== undefined &&
```

Comment:
I think this is the wrong place b/c `getRenderStyle` is called at decoding time but especially for interpolated properties the size and opacity could get zero at any time.

Reply:
I also don't think we should modify the `backgroundOpacity`. Instead there should be a check when the label is about to be rendered (e.g. in `TextElementsRenderer`) whether `backgroundSize > 0  && backgroundOpacity > 0`